### PR TITLE
chore(rpa): add first documentation for RPA bridge

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-12169

### CODE REVIEW

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    name: "Camunda BPM RPA Bridge"
+    identifier: "installation-camunda-bpm-rpa-bridge"
+    parent: "installation-guide"
+    pre: ""
+---
+
+This page describes the steps to orchestrate and execute RPA bots from Camunda BPM using the Camunda BPM RPA Bridge.
+
+{{< enterprise >}}
+  Please note that the RPA bridge is only available as enterprise edition.
```
Comment (koevskinikola) [inline content/installation/camunda-bpm-rpa-bridge.md:17]: I think this should state that the bridge is available "in" or "through" the EE version. Or for EE customers.
```suggestion
  Please note that the RPA bridge is only available in the Enterprise Edition.
```

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Java
+Please make sure that you have the Java Runtime Environment 8+ installed.
+
+You can verify this by using your terminal, shell, or command line:
+
+```sh
+java -version
+```
+If you need to install Java Runtime Environment, you can [find the download from Oracle here](https://www.oracle.com/java/technologies/javase-downloads.html).
```
Comment (koevskinikola) [inline content/installation/camunda-bpm-rpa-bridge.md:30]:
```suggestion
If you need to install a Java Runtime Environment, you can [find the download from Oracle here](https://www.oracle.com/java/technologies/javase-downloads.html).
```

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Camunda BPM
+You will need a running Camunda BPM Enterprise Edition with Camunda 7.14 or later. You can find all downloadable distros [here](https://downloads.camunda.cloud/enterprise-release/camunda-bpm/).
+
+## Cawemo and Camunda Modeler
+The easiest way to create a BPMN model that connects to an RPA bot is by using Cawemo to create a worker catalog that you can apply to your process model using the Camunda Modeler. Make sure to use Cawemo 1.4 (or later) and Camunda Modeler 4.2 (or later).
+
+# Installation Procedure
+1. Downlaod the Camunda RPA Bridge distro from [here](). TODO
```
Comment (koevskinikola) [inline content/installation/camunda-bpm-rpa-bridge.md:39]: Typo. Also the download link needs to be added.
```suggestion
1. Download the Camunda RPA Bridge distro from [here](). TODO
```

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+1. Extract the downloaded archive
+1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}}) for a full list of configuration options).
+1. Start the Bridge by executing the jar file: `java -jar camunda-bpm-rpa-bridge.jar`
+1. create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}} for more information)
```
Comment (koevskinikola) [inline content/installation/camunda-bpm-rpa-bridge.md:43]: Fix typo. We can also further specify the section of the docs.
```suggestion
1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}} for more information)
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+menu:
+  main:
+    identifier: "camunda-bpm-rpa-bridge"
+    parent: "user-guide"
+
+---
+
+The Camunda RPA bridge is a standalone application that allows to call RPA (robotic process automation) bots from BPMN models deployed to a Camunda engine. RPA bots can be orchestrated as External Tasks using the Camunda Modeler and Cawemo.
+
+For detailed instructions on how to connect your first RPA bot to a BPMN model (using Cawemo and the Camunda Modeler) and execute it from a running process instnace using the Camunda engine head over to our [installation guide]({{< ref "/installation/camunda-bpm-rpa-bridge.md" >}})
```
Comment (koevskinikola) [inline content/user-guide/camunda-bpm-rpa-bridge.md:15]: Typo
```suggestion
For detailed instructions on how to connect your first RPA bot to a BPMN model (using Cawemo and the Camunda Modeler) and execute it from a running process instance using the Camunda engine, head over to our [installation guide]({{< ref "/installation/camunda-bpm-rpa-bridge.md" >}})
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+{{< enterprise >}}
+  Please note that the RPA bridge is only available as enterprise edition.
+{{< /enterprise >}}
+
+# The Basics
+
+The Camunda RPA bridge serves a connector between Camunda (BPMN) on the one side and UIPath (RPA) on the other side. Processes running inside the Camunda engine can define external tasks that are marked as RPA tasks.
```
Comment (koevskinikola) [inline content/user-guide/camunda-bpm-rpa-bridge.md:23]: Fix typo. Also, should we provide a link to the `UIPath` website?
```suggestion
The Camunda RPA bridge serves as a connector between Camunda (BPMN) on the one side and UIPath (RPA) on the other side. Processes running inside the Camunda engine can define external tasks that are marked as RPA tasks.
```

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+## Camunda BPM
+You will need a running Camunda BPM Enterprise Edition with Camunda 7.14 or later. You can find all downloadable distros [here](https://downloads.camunda.cloud/enterprise-release/camunda-bpm/).
+
+## Cawemo and Camunda Modeler
+The easiest way to create a BPMN model that connects to an RPA bot is by using Cawemo to create a worker catalog that you can apply to your process model using the Camunda Modeler. Make sure to use Cawemo 1.4 (or later) and Camunda Modeler 4.2 (or later).
+
+# Installation Procedure
+1. Downlaod the Camunda RPA Bridge distro from [here](). TODO
+1. Extract the downloaded archive
+1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}}) for a full list of configuration options).
```
Comment (koevskinikola) [inline content/installation/camunda-bpm-rpa-bridge.md:41]: We can further specify the section of the User Guide docs:
```suggestion
1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#camunda-bpm-rpa-bridge-configuration" >}}) for a full list of configuration options).
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+    <td>Relative path of the webhook endpoint in your application.</td>
+    <td>/webhook/event</td>
+  </tr>
+</table>
+
+# Set up an RPA Task
+
+The RPA bridge is a regular Java external task client and RPA tasks are [external tasks]({{< ref "/user-guide/process-engine/external-tasks.md" >}}) with specific settings.
+
+By default the bridge listens for tasks with the topics `uipath` and `RPA`. If a process instance reaches an external task with one of these topics, the bridge will be able to fetch and lock them. Once a task is locked the bridge will try to forward it to your installation of UIPath which will take care of executing the associated robots.
```
Comment (koevskinikola) [inline content/user-guide/camunda-bpm-rpa-bridge.md:149]:
```suggestion
By default, the bridge listens for tasks with the topics `uipath` and `RPA`. If a process instance reaches an external task with one of these topics, the bridge will be able to fetch and lock them. Once a task is locked, the bridge will try to forward it to your installation of UIPath which will take care of executing the associated robots.
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+# The Basics
+
+The Camunda RPA bridge serves a connector between Camunda (BPMN) on the one side and UIPath (RPA) on the other side. Processes running inside the Camunda engine can define external tasks that are marked as RPA tasks.
+
+The bridge extends the regular [Java External Task Client](https://github.com/camunda/camunda-external-task-client-java), fetches and locks the RPA tasks and starts a job in UIPath. Once the job is done, UIPath will notify a webhook in the bridge about the job result and state (Success/Failure). The bridge will complete the previously locked external task and pass any result variables received from UIPath along to the Camunda engine.
+
+# Prerequisites and Supported Environments
+The Camunda RPA Bridge supports the on-premise version of the [UIPath Orchestrator](https://www.uipath.com/product/orchestrator). To execute RPA bots from Camunda you will need:
+
+* An on-premise installation of [UIPath Orchestrator](https://www.uipath.com/product/orchestrator).
```
Comment (koevskinikola) [inline content/user-guide/camunda-bpm-rpa-bridge.md:30]: Making the bullet points consistent:
```suggestion
* An on-premise installation of [UIPath Orchestrator](https://www.uipath.com/product/orchestrator)
```

File: content/installation/camunda-bpm-rpa-bridge.md (hunk 1 above)
Comment (mboskamp) [reply content/installation/camunda-bpm-rpa-bridge.md:17]: I don't think this change is necessary as the bridge will not be bundled within something else (_in_ the Enterprise Edition). It is an Enterprise Edition itself and there is no Community Edition.

What do you think about this?
```suggestion
  Please note that the RPA bridge is only available for customers with access to our enterprise artifacts.
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+# Set up an RPA Task
+
+The RPA bridge is a regular Java external task client and RPA tasks are [external tasks]({{< ref "/user-guide/process-engine/external-tasks.md" >}}) with specific settings.
+
+By default, the bridge listens for tasks with the topics `uipath` and `RPA`. If a process instance reaches an external task with one of these topics, the bridge will be able to fetch and lock them. Once a task is locked, the bridge will try to forward it to your installation of UIPath which will take care of executing the associated robots.
+
+## Map a Task to an RPA Bot
+
+To tell the bridge for which RPA process a bot should be started, it is necessary to add a [custom extension]({{< ref "/user-guide/model-api/bpmn-model-api/extension-elements.md" >}}) with the name `bot` and a value equal to a released process in UIPath.
```
Comment (volkergersabeck) [inline content/user-guide/camunda-bpm-rpa-bridge.md:153]: "value equal to the package name of the released process in UIPath"

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+
+The extension property can be set using the Camunda Modeler (or any other BPMN or XML editor). In the Modeler, select the task and switch to the Extensions tab in the properties panel. Add a property named `bot` with the value equal to the name of the process you want to start in UIPath.
+
+This example shows an external task that will trigger the UIPath process `Launch the Robots` and is published in the `RPA` external task topic.
+
+```xml
+<bpmn:serviceTask id="myRPAtask" name="Launch the Robots" camunda:type="external" camunda:topic="RPA">
+  <bpmn:extensionElements>
+    <camunda:properties>
+      <camunda:property name="bot" value="UIPathProcessName" />
```
Comment (volkergersabeck) [inline content/user-guide/camunda-bpm-rpa-bridge.md:169]: value="UIPathPackageName" />

File: content/installation/camunda-bpm-rpa-bridge.md (hunk 1 above)
Comment (koevskinikola) [reply content/installation/camunda-bpm-rpa-bridge.md:17]: Looks good! :+1:

**Review by koevskinikola — APPROVED** [review]
From my side, as soon as the remaining suggestions are resolved, the PR can be merged.

**Comment by mboskamp** [comment]
Thanks @koevskinikola, thanks @volkergersabeck. I will leave the PR open, because we will only release it when the bridge will be released. Until then feel free to add your suggestions to this PR.

File: content/installation/camunda-bpm-rpa-bridge.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+## Cawemo and Camunda Modeler
+The easiest way to create a BPMN model that connects to an RPA bot is by using Cawemo to create a worker catalog that you can apply to your process model using the Camunda Modeler. Make sure to use Cawemo 1.4 (or later) and Camunda Modeler 4.2 (or later).
+
+# Installation Procedure
+1. Download the Camunda RPA Bridge distro from [here](). TODO
+1. Extract the downloaded archive
+1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#camunda-bpm-rpa-bridge-configuration" >}}) for a full list of configuration options).
+1. Start the Bridge by executing the jar file: `java -jar camunda-bpm-rpa-bridge.jar`
+1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}} for more information)
```
Comment (tmetzke) [inline content/installation/camunda-bpm-rpa-bridge.md:43]:
```suggestion
1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}}) for more information)
```

File: content/user-guide/camunda-bpm-rpa-bridge.md [hunk 13]
```diff
@@ -0,0 +1,377 @@
...
+## Enable Telemetry
+
+<table class="table desc-table">
+  <tr>
+    <td rowspan="15"><code>org.camunda.bpm.rpa</code></td>
+      <td><code>enable-telemetry</code></td>
+      <td>Enable sending usage metrics and general statistics to Camunda. For more information see the <a href="#telemetry">Telemetry section</a> below.</td>
+      <td>false</td>
+  </tr>
+</table>
```
Comment (tmetzke) [inline content/user-guide/camunda-bpm-rpa-bridge.md:270]: This should probably simply go into `Configure Basic Properties`. In order to reference this somewhere, you will need to add an `id` here in the `<tr>` as well, don't you?

**Review by tmetzke — CHANGES_REQUESTED** [review]
Looks good overall 👍 
Just a minor thing for the telemetry config option

File: content/user-guide/camunda-bpm-rpa-bridge.md (hunk 13 above)
Comment (mboskamp) [reply content/user-guide/camunda-bpm-rpa-bridge.md:270]: Will move the section up. Not sure about the id. You can reference the section with the header name (#enable-telemetry).

Comment (tmetzke) [reply content/user-guide/camunda-bpm-rpa-bridge.md:270]: Ah ok, no I meant to simply integrate the config option into the other section, next to the license key.
And then add an id to the `tr` and reference this from the `Telemetry` section

Comment (mboskamp) [reply content/user-guide/camunda-bpm-rpa-bridge.md]: Got it. Makes sense.

**Review by tmetzke — APPROVED** [review]

**Pull request merged**